### PR TITLE
ticket 5 complete

### PR DESCRIPTION
**Pull request by yassmine-x**

### CODE REVIEW

File: __tests__/news.test.js [hunk 1]
```diff
@@ -67,3 +67,69 @@ describe("ERRORS FOR /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("5. PATCH /api/articles/:article_id", () => {
```
Comment (daviidbartlett) [inline __tests__/news.test.js]: remove .only

File: __tests__/news.test.js [hunk 2]
```diff
@@ -67,3 +67,69 @@ describe("ERRORS FOR /api/articles/:article_id", () => {
...
+      .patch("/api/articles/1")
+      .send(votesChange)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request!");
+      });
+  });
+});
+
+describe.only("test", () => {});
```
Comment (daviidbartlett) [inline __tests__/news.test.js]: not needed

File: controllers/topics.controller.js [hunk 3]
```diff
@@ -2,6 +2,9 @@ const fetchTopics = require("../models/topics.model");
 
 const getTopics = (req, res) => {
   fetchTopics().then((topics) => {
+    if (Object.keys(topics) === 0) {
```
Comment (daviidbartlett) [inline controllers/topics.controller.js]: why had this been added in this ticket?

File: models/changeVotes.model.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+const updateVotes = (id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id=$1", [id])
```
Comment (daviidbartlett) [inline models/changeVotes.model.js]: this isn't updating the article. It's selecting the article.

Comment (daviidbartlett) [reply models/changeVotes.model.js]: your db, isn't actually being updated.

**Review by HeatherEver — APPROVED** [review]

**Pull request merged**